public/static/locales/en/common.json

```diff
@@ -451,9 +451,11 @@
   "url": "URL",
   "hidden": "Hidden",
   "readonly": "Readonly",
+  "plan_description": "You're currently on the {{plan}} plan.",
+  "plan_upgrade_invitation": "Upgrade your account to the pro plan to unlock all of the features we have to offer.",
   "plan_upgrade": "You need to upgrade your plan to have more than one active event type.",
   "plan_upgrade_teams": "You need to upgrade your plan to create a team.",
-  "plan_upgrade_instructions": "To upgrade, go to <a href=\"https://cal.com/upgrade\" className=\"underline\">https://cal.com/upgrade</a>",
+  "plan_upgrade_instructions": "You can <1>upgrade here</1>.",
```

Comment:
Does this work?

Reply:
Yup. Html in there doesn't. But this does

Reply:
it doesn't seem to, clicking just redirects to the homepage

Reply:
You need to be running both calendso and the website. Since the upgrade logic is in there.